executor/partition_table_test.go

```diff
@@ -701,52 +701,52 @@ func TestOrderByAndLimit(t *testing.T) {
 		// check if tiflash is used
 		require.True(t, tk.HasTiFlashPlan(queryPartitionWithTiFlash), fmt.Sprintf("%v", tk.MustQuery("explain "+queryPartitionWithTiFlash).Rows()))
 		// but order is not pushed
-		require.False(t, tk.HasPlan(queryPartitionWithTiFlash, "Limit"), fmt.Sprintf("%v", tk.MustQuery("explain "+queryPartitionWithTiFlash).Rows()))
+		tk.MustNotHavePlan(queryPartitionWithTiFlash, "Limit")
```

Comment:
Should the message part also be kept, or should the message in Must[Not]HavePlan also include the full Explain, to see the full plan as well as what was expected?